frontend/src/app/settings/core/user-settings/user-settings.component.html

```diff
@@ -22,25 +22,52 @@ <h2>Manage users</h2>
       [class]="
         'scrollable simple-scroll' // eslint-disable-line tailwindcss/no-custom-classname
       "
-      (selectionChange)="onUserSelect($event.options[0].value)"
     >
       <div mat-subheader>Administrators</div>
-      <mat-list-option
-        *ngFor="let user of getUsersByRole('administrator')"
-        [value]="user"
-      >
-        <mat-icon mat-list-icon>account_circle</mat-icon>
-        <div mat-line>{{ user.name }}</div>
-        <div mat-line>{{ projectUserService.ADVANCED_ROLES[user.role] }}</div>
+      <mat-list-option *ngFor="let user of getUsersByRole('administrator')">
```

Comment:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/20246db0-c00c-49c3-9272-cf8ecc6ad394)

Instead of this unintuitive approach (admins have to click on the info-symbol) to upgrade or downgrade users and overflow of the text in the icon, keep it simple: 

- Implement the same design like in the project details here.
- Move the user information to the user profile page, only visible for admins.

Reply:
what does your second point mean? as a admin doesn't it make sense to see whether it is a user or a administrator? or do you want to have that indirectly displayed via the up and downgrade arrows?

Reply:
in addition I would leave the "i" symbol as otherwise we are losing the user information right?

Reply:
> in addition I would leave the "i" symbol as otherwise we are losing the user information right?

No, we don't need it. Check the second point: Move the user information to the user profile page, only visible for admins. It should just be part of the user profile page, not the user settings anymore. To see it, the admin would open the profile page.

Reply:
> what does your second point mean? as a admin doesn't it make sense to see whether it is a user or a administrator? or do you want to have that indirectly displayed via the up and downgrade arrows?

I don't understand the question. The role is indicated below the username: 
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/304ab5e4-6b9c-4084-b679-8da12ca88394)

Reply:
I see - thanks for the clarification 
I did not get what you meant